workflows/files/ZWEWRF06.xml

```diff
@@ -0,0 +1,251 @@
+        <string valueMustBeChoice="false" multiLine="false">
+            <validationType>DSNAME</validationType>
+        </string>
+    </variable>
+    <variable name="xmem_prefix" scope="instance" visibility="public">
+        <label>Cross Memory Server DSN Prefix</label>
+        <abstract>Please specify the HLQ that will be used for the XMEM</abstract>
+        <description>High Level Qualifier that will be used to install and configure the Cross Memory Server</description>
+        <category>Installation</category>
+        <string valueMustBeChoice="false" multiLine="false">
+            <validationType>DSNAME</validationType>
+        </string>
+    </variable>
+    <variable name="xmem_dasd" scope="instance" visibility="public">
+        <label>Cross Memory Server DASD</label>
+        <abstract>Please specify the DASD that will be used for the XMEM data sets.</abstract>
+        <description>Please specify the DASD that will be used for the XMEM data sets.
+Specify the DASD in either of the following formats:
+VOL=SER=&lt;i&gt;volume&lt;/i&gt; for non-SMS managed data sets
+STORCLAS=&lt;i&gt;storage_class&lt;/i&gt; for SMS managed data sets</description>
+        <category>Installation</category>
+        <string valueMustBeChoice="false" multiLine="false">
+            <choice>VOL=SER=??????</choice>
+            <choice>STORCLAS=????????</choice>
+        </string>
+    </variable>
+    <variable name="proclib" scope="instance" visibility="public">
+        <label>Procedure Library</label>
+        <abstract>Please specify the procedure library for the started task</abstract>
+        <description>Specify the PROCLIB data set name that will be used to store the XMEM STC JCL</description>
+        <category>Installation</category>
+        <string valueMustBeChoice="false" multiLine="false">
+            <validationType>DSNAME</validationType>
+        </string>
+    </variable>
+    <variable name="parmlib" scope="instance" visibility="public">
+        <label>Parameter Library</label>
+        <abstract>Please specify the PARMLIB of your active SCHEDxx member</abstract>
+        <description>Please specify the active system PARMLIB where your active SCHEDxx member resides in.</description>
+        <category>Parmlib</category>
+        <string valueMustBeChoice="false" multiLine="false">
+            <validationType>DSNAME</validationType>
+        </string>
+    </variable>
+    <variable name="sched" scope="instance" visibility="public">
+        <label>Scheduler member</label>
+        <abstract>Please specify the name of the active SCHEDxx member</abstract>
+        <description>Please specify the scheduler member that is currently used in the specified PARMLIB.</description>
+        <category>Parmlib</category>
+        <string valueMustBeChoice="false" multiLine="false">
+            <validationType>DSMEMBERNAME</validationType>
+            <default>SCHED00</default>
+        </string>
+    </variable>
```

Comment:
I am putting the ZWESIP00 to the `${instance-zowe_dsn_prefix}`.PARMLIB by default, should that be changeable?

Reply:
Sorry, I missed this question.
Using ${instance-zowe_dsn_prefix} is a good default to place sample PARMLIB updates for the manual path. I don't think customers will ask to make this a variable, as long as it's obvious to them where the samples are placed. Personally I would use `@.PARMLIB` as default LLQ, so that the customized datasets stand out (and are alphabetically on top) among the product datasets, but I grew up in environments with a lot more product datasets than Zowe's 2.
Do we still have an automatic path as well, or did that get cancelled?